src/fragments/lib/push-notifications/react-native/getting_started/40_install_lib.mdx

```diff
@@ -0,0 +1,14 @@
+In your application directory, you should first install the necessary dependencies for using Amplify Push Notifications.
+
+```bash
+npm install aws-amplify @aws-amplify/rtn-push-notification amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill
+```
+
+You need to add the crypto.getRandomValues and URL polyfill to your application's entry point file (in most React Native apps this will be the top level `index.js`).
```

Comment:
```suggestion
You will also need to add the crypto.getRandomValues and URL polyfills to your application's entry point file (in most React Native apps this will be the top level `index.js`).
```